use-timescale/services/dynamic-postgresql.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Dynamic PostgreSQL
+excerpt: How Dynamic PostgreSQL works
+products: [cloud]
+keywords: [services, create, postgresql, dynamic]
+cloud_ui:
+    path:
+        - [services]
+        - [create_services]
+---
+
+# Dynamic PostgreSQL
+
+Dynamic PostgreSQL is a managed PostgreSQL offering on Timescale that is built 
+for your production workloads. Dynamic PostgreSQL is 100% PostgreSQL with no 
+alterations to the core database. It comes with Timescale's dynamic compute and 
+usage-based storage, giving you the scalability of a serverless database with 
+the performance and cost-effectiveness of a statically allocated DBaaS.
+
+## Dynamic compute
+
+With Dynamic PostgreSQL and dynamic compute, you choose a compute range with a 
+minimum and maximum. Your database always has the minimum compute 
```

Comment:
```suggestion
Dynamic compute means you choose a compute range with a 
minimum and maximum instead of a single static compute option. Your database always has the minimum compute 
```